third_party/gems/bundle-env.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# --- begin runfiles.bash initialization ---
+# Copy-pasted from Bazel's Bash runfiles library (tools/bash/runfiles/runfiles.bash).
```

Comment:
Ultimately what's the purpose of this snippet?

My guess is that it's basically just here to allow for `rlocation` to be a function that people can call, is that right?

Maybe we should add a link here:

https://github.com/bazelbuild/bazel/blob/master/tools/bash/runfiles/runfiles.bash

at the top, or better yet link to a commit sha where you copied this snippet from?

Reply:
(or if you modified it that's fine too, just explain that)

Reply:
I'm happy to include a link to the original file. It's just there because the environment that your script is run in can be wildly different depending on the context it's run from (another script vs through `bazel run` etc.).

Reply:
I added links to the version we're using with 0.21.0.